audformat/core/testing.py

```diff
@@ -320,11 +320,42 @@ def to_scheme_type(series: pd.Series) -> str:
         labels={1: 'a', 2: 'b', 3: 'c'}
     )
     db.schemes['label_map_str'] = Scheme(
-        labels={'label1': {'prop1': 1, 'prop2': 'a'},
-                'label2': {'prop1': 2, 'prop2': 'b'},
-                'label3': {'prop1': 3, 'prop2': 'c'}})
+        labels={
+            'label1': {'prop1': 1, 'prop2': 'a'},
+            'label2': {'prop1': 2, 'prop2': 'b'},
+            'label3': {'prop1': 3, 'prop2': 'c'},
+        }
+    )
     db.schemes['string'] = Scheme()
     db.schemes['time'] = Scheme(dtype=define.DataType.TIME)
+    db.schemes['age'] = Scheme(
+        dtype=define.DataType.INTEGER,
+        minimum=9,
+        maximum=99,
+    )
+    db.schemes['gender'] = Scheme(
+        labels=['female', 'male'],
+    )
+
+    ##############
+    # Misc Table #
+    ##############
+
+    index = pd.Index(
+        ['spk1', 'spk2', 'spk3'],
+        name='speaker',
+    )
+    db['misc'] = MiscTable(index)
+    db['misc']['age'] = Column(scheme_id='age')
+    db['misc']['age'].set(db.schemes['age'].draw(len(index)))
+    db['misc']['gender'] = Column(scheme_id='gender')
+    db['misc']['gender'].set(db.schemes['gender'].draw(len(index)))
+
+    ############################
+    # Schemes from Misc Tables #
+    ############################
+
+    db.schemes['speaker'] = Scheme(labels='misc', dtype='str')
```

Comment:
The other schemes we call after the data type, so maybe we should rename to `label_from_misc`?

Reply:
On the other hand, this makes sense since we again map the columns of `map` to `age` and `gender`....

Reply:
I renamed to `label_map_misc` and use `label` and `int` inside the corresponding misc table.

As we need `label` also for another test as a column in the table, we still assign it to the tables, also you would not expect it for a misc scheme as you would use mapping to get it.